docs/tutorials/getting-started-with-node-js.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Getting Started with Node JS
+group: software-languages
+author: "@liam-b"
+---
+
+{% include /style/icon.html type="warning" %}
+Note that Node JS run really slowly on the EV3 brick and is limited to NPM version 0.10.29 which can be a problem when trying to install some libraries.
+{: .alert .alert-warning}
+
+**Before you start, make sure that you have configured a network connection to
+your ev3dev device and have opened an SSH connection to it.**
+
+## Installing ev3dev-lang
+To install the library run:
+
+```shell
+npm install ev3dev-lang
+```
+
+Now whenever you want to use this in code, just write:
+
+```javascript
+var ev3dev = require('ev3dev-lang');
+```
+
+For a bit more info on this visit the ev3dev-lang [github page](https://github.com/wasabifan/ev3dev-lang-js).
+
+## Nano
+For this tutorial, we will be using nano. For more info on how to use nano check out the [nano cheat sheet](/docs/tutorials/nano-cheat-sheet).
+
+## Hello world 
+Navigate to a project directory (e.g `~/src/js`) and make a new file with `touch test.js`
+
+Open it with `nano test.js` and write:
+```javascript
+var test = 'hello world!';
+console.log(test);
+```
+
+Running `node test.js`should print `hello world!` to the console. If this worked you are ready to start coding with node!
```

Comment:
Although I've often formatted the name as "node" or "Node," they [officially use "Node.js"](https://nodejs.org/en/); I think we should align with that wherever possible.